src/components/Header/Button/DarkMode/DarKMode.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useContext } from 'react';
+import { ThemeContext } from '../../../../providers/Theme/Theme.provider';
+import { DMBtn } from './DarkMode.styled';
+
+function MyDarkMode(props) {
+  const mode = useContext(ThemeContext)
+  const {darkMode} = mode.state
+  
+  const str = props.theme;
+  const modeText = str.charAt(0).toUpperCase() + str.slice(1);
+
+  const handleDarkMode = () => {
+    if (darkMode === 'dark') {
+      mode.dispatch({ type: "LIGHTMODE" });
+    } else {
+      mode.dispatch({ type: "DARKMODE" });
+    }
+  };
+
+  return (
+    <DMBtn theme={props.theme}>

```

Comment:
```suggestion
    <DMBtn theme={theme}>
```
If you implement the first suggestion